roles/automate/files/bbautomate.py

```diff
@@ -38,11 +37,20 @@ class bcolors:
 
 def load_file(filename):
     logging.info(bcolors.OKBLUE+'Loading '+filename+bcolors.ENDC)
+
+    # Select YAML loader (needs PyYAML 5.1+ to be safe)
+    yaml_version_major=int(yaml.__version__.split('.')[0])
+    yaml_version_minor=int(yaml.__version__.split('.')[1])
+    if yaml_version_major > 5 or (yaml_version_major == 5 and yaml_version_minor >= 1):
+        yaml_loader='Safe'
+    else:
+        yaml_loader='Unsafe'
+
     with open(filename, 'r') as f:
-        if version.parse(yaml.__version__) < version.parse("5.1.0"):
-            return yaml.load(f)
-        else:
+        if yaml_loader == 'Safe':
```

Comment:
Following on from previous comment:

```
if yaml_loader_safe:
    return yaml.load(f, Loader=yaml.FullLoader)
return yaml.load(f)
```

Note: the `else` is not needed due to the first `return`